web/server/vue-cli/src/components/Report/ReportStepMessage.vue

```diff
@@ -73,7 +79,8 @@ export default {
     index: { type: [ Number, String ], default: null },
     bus: { type: Object, default: null },
     prevStep: { type: Object, default: null },
-    nextStep: { type: Object, default: null }
+    nextStep: { type: Object, default: null },
+    docUrl: { type: String, default: "" }
```

Comment:
This is a minor comment, but "not having some kind of information" could be represented by `null` like in case of the other members. `docUrl` could be changed accordingly in this `ReportStepMessage.vue` and `Report.vue` file tool.